packages/cli/src/ActiveWorkflowRunner.ts

```diff
@@ -348,30 +345,31 @@ export class ActiveWorkflowRunner {
 	/**
 	 * Returns the ids of the currently active workflows
 	 */
-	async getActiveWorkflows(user?: User): Promise<IWorkflowDb[]> {
+	async getActiveWorkflows(user?: User): Promise<string[]> {
```

Comment:
How about 
```suggestion
	async getActiveWorkflowIds(user?: User): Promise<IWorkflowDb['id'][]> {
```

Reply:
that'd be an `{id:string}` object instead of string[], which for some reason the frontend expects